src/topology/continuous_function/basic.lean

```diff
@@ -77,35 +118,47 @@ def equiv_fn_of_discrete [discrete_topology α] : C(α, β) ≃ (α → β) :=
 
 end
 
+variables (α)
+
 /-- The identity as a continuous map. -/
-def id : C(α, α) := ⟨id⟩
+protected def id : C(α, α) := ⟨id⟩
```

Comment:
Why not use `@[simps]` instead of manually defining `coe_id`? (Also for `const` and `comp`.)
```suggestion
@[simps coe {fully_applied := ff}]
protected def id : C(α, α) := ⟨id⟩
```

Reply:
Yes please!

Reply:
Okay, so the problem is that `simps` seems to only be able to add the prefix to the end of the name, which gives `id_coe`, not `coe_id`. Further, it only generates the projection, not the applied projection. So
we get
```
@[simp] lemma id_apply : ⇑(continuous_map.id α) = id := rfl
```
instead of
```
@[simp] lemma id_apply (a : α) : continuous_map.id α a = a := rfl
```
Long story short, it doesn't do what I want.

Reply:
Alright, then we'll just do it manually. (Until `simps` becomes smart enough.)